config/locales/fr.yml

```diff
@@ -93,6 +110,9 @@ fr:
     follow:
       body: "%{name} vous suit !"
       subject: "%{name} vous suit"
+    follow_request:
+      body: "%{name} a demandé à vous suivre"
+      subject: 'Abonné⋅es en attente : %{name}'
```

Comment:
> If we're going there with ``:``, then it should be done for ``:;!?``.

Actually, if we're talking about french from France, [yes](https://fr.wikipedia.org/wiki/Ponctuation#En_fran.C3.A7ais), we should.